### DISCUSSION TITLE
How to remove suffix from custom module?

### DISCUSSION THREAD

**Discussion by ollebergkvist** [starship/starship#6375] [Q&A] [answered]
**Config:**
[custom.doppler]
command = "doppler configure get config --plain"
when = "which doppler"
symbol = "🔐 "
style = "bold blue"
format = "via [$symbol($output)]($style)"

**Current result:**
dir/apps/api on  feature/dol-2205-create-seed-tokens-script via ⬢ v20.17.0 via 🔐 prd.doppler via 🐳

**Desired result:**
dir/apps/api on  feature/dol-2205-create-seed-tokens-script via ⬢ v20.17.0 via 🔐 prd via 🐳

Without ".doppler"

**Comment by davidkna** [comment, marked as answer]
Are you using `$custom.doppler` instead of `${custom.doppler}` in your global `format` string? The braces are required due to the dot and `$custom` is a shortcut for all not-explicitly included custom modules.

**Reply by ollebergkvist** [reply]
Thank you!
